pkg/resources/external_table_grant_acceptance_test.go

```diff
@@ -0,0 +1,74 @@
+package resources_test
+
+import (
+	"bytes"
+	"strings"
+	"testing"
+	"text/template"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAccExternalTableGrantFuture(t *testing.T) {
```

Comment:
```suggestion
func TestAccExternalTableGrant_future(t *testing.T) {
```

Reply:
So I was looking at other examples of format for test names and the suggestion above conflicts with others.  I ended up with the name `TestAcc_ExternalTableFutureGrant` and applied that elsewhere.  Let me know if you'd like something different.